build.gradle.kts

```diff
@@ -66,20 +66,22 @@ subprojects {
     resolutionStrategy.force(groovy.util.Eval.x(this@subprojects, "x.dep.errorProneCore"))
   }
 
-  // TODO: Make this lazy
-  this@subprojects.afterEvaluate {
-    val jarTask = this@subprojects.tasks.findByName("jar") as? org.gradle.jvm.tasks.Jar
-    if (jarTask != null) {
-      val japiCmp = this@subprojects.tasks.register("japicmp", me.champeau.gradle.japicmp.JapicmpTask::class.java) {
-        oldClasspath = this@subprojects.files(baselineJar(this@subprojects, "1.2.1"))
-        newClasspath = this@subprojects.files(jarTask.archiveFile)
-        ignoreMissingClasses = true
-        packageExcludes = listOf("*.internal*")
-        onlyModified = true
-        txtOutputFile = this@subprojects.file("$buildDir/reports/japi.txt")
-      }
-      rootJapiCmp.configure {
-        dependsOn(japiCmp)
+  if (name != "apollo-compiler" && name != "apollo-gradle-plugin-incubating" && name != "apollo-coroutines-support") {
```

Comment:
I had to disable registration of `japicmp` task for these modules, otherwise build fails with this error:
```
Execution failed for task ':apollo-android:apollo-compiler:compileKotlin'.
> Could not create task ':apollo-android:apollo-compiler:japicmp'.
   > Collection contains no element matching the predicate.
```
locally (weird that at the same time it seems its ok on CI).

Reply:
Can you run from command line with --stacktrace option with more details?

Reply:
I've seen this during development but it's working well here, even on this branch. Maybe it's due to the publishing configuration that creates some extra 'jar' tasks if you have things like BINTRAY_USER defined ?